packages/babel-plugin-polyfill-corejs2/src/built-in-definitions.ts

```diff
@@ -1,36 +1,37 @@
-// @flow
-
 import corejs2Polyfills from "@babel/compat-data/corejs2-built-ins";
 
-type ObjectMap<V> = { [name: string]: V };
+type ObjectMap<V> = {
+  [name: string]: V;
+};
 
 type PolyfillDescriptor<T> = {
-  name: string,
-  pure: ?string,
-  global: string[],
-  meta: ?T,
+  name: string;
+  pure: string | undefined | null;
+  global: string[];
+  meta: T | undefined | null;
 };
 
 type CoreJS2Meta = {
-  minRuntimeVersion: ?string,
+  minRuntimeVersion: string | undefined | null;
 };
 
 const define = <T>(
   name: string,
-  pure: ?string,
+  pure?: string | null,
   global: string[] = [],
-  meta: ?T,
+  meta?: T | null,
 ): PolyfillDescriptor<T> => {
   return { name, pure, global, meta };
 };
 
 const pureAndGlobal = (
   pure: string,
   global: string[],
-  minRuntimeVersion: ?string = null,
+  minRuntimeVersion: string | undefined | null = null,
 ) => define<CoreJS2Meta>(global[0], pure, global, { minRuntimeVersion });
 
-const globalOnly = (global: string[]) => define(global[0], null, global);
+const globalOnly = (global: string[]) =>
+  define<CoreJS2Meta>(global[0], null, global);
```

Comment:
missing generic here caused a TS error. guess it doesn't error in Flow for some reason?